src/atomate2/cli/dev.py

```diff
@@ -379,9 +387,15 @@ def _fake_dirdata(
                 allow_missing=allow_missing,
             )
         if include_fake_files:
-            for fname in include_fake_files:
+            for fn in include_fake_files:
+                fname = check_file_ext_pert(filename=fn, dirname=src_dirdata)
```

Comment:
I understood the point of this check and the `remove_copied_out_ddb` below. But it took me a while. Can you add a few comments to clarify the need for these changes for future reference?

Reply:
sure thing